tests/test_resample.py

```diff
@@ -1,26 +1,34 @@
+import random
+
 import numpy as np
 import pytest
 
 from diffpy.utils.resampler import wsinterp
 
 
 def test_wsinterp():
-    import random
-
-    # Check known points are unchanged by interpolation
     # FIXME: if another SW interp function exists, run comparisons for interpolated points
+
     # Sampling rate
     ssr = 44100**-1  # Standard sampling rate for human-hearable frequencies
-    t = ssr
+
+    # Creating a symmetric set of sample points around zero.
     n = 5
-    xp = np.array([i * t for i in range(-n, n + 1, 1)])
-    x = np.array([i * t for i in range(-n - 1, n + 2, 1)])
+    xp = np.array([i * ssr for i in range(-n, n + 1, 1)])
+    x = np.array([i * ssr for i in range(-n - 1, n + 2, 1)])
+    assert len(xp) == 11 and len(x) == 13
 
-    # Interpolate a few random datasets
+    # Generating fp values across 10 trial runs
     trials = 10
-    for trial in range(trials):
-        fp = np.array([random.random() * ssr for i in range(-n, n + 1, 1)])
+
+    for _ in range(trials):
+        # Create random function values (fp) at the points defined in xp above
+        fp = np.array([random.random() * ssr for _ in range(-n, n + 1, 1)])
+
```

Comment:
Let's close this up. No need for the blank lines that are not adding subtracting value.  This is true elsewhere in this file too